Tests are failing on arm and aarch64

```
    def checkNotifs(self):
>       assert self.client_factory.notifs == 1
E       AssertionError: assert 0 == 1
E        +  where 0 = <autobahn.twisted.websocket.WebSocketClientFactory object at 0x7f78dc6c70>.notifs
E        +    where <autobahn.twisted.websocket.WebSocketClientFactory object at 0x7f78dc6c70> = <test_wallet_rpc.TrialTestWRPC_WS testMethod=test_notif>.client_factory

jmclient/test/test_wallet_rpc.py:125: AssertionError
```

Same happens on x86_64.

```
============================================================= FAILURES =============================================================
___________________________________________________ TrialTestWRPC_WS.test_notif ____________________________________________________

self = <test_wallet_rpc.TrialTestWRPC_WS testMethod=test_notif>

    def checkNotifs(self):
>       assert self.client_factory.notifs == 1
E       assert 0 == 1
E        +  where 0 = <autobahn.twisted.websocket.WebSocketClientFactory object at 0x7f80e43e9090>.notifs
E        +    where <autobahn.twisted.websocket.WebSocketClientFactory object at 0x7f80e43e9090> = <test_wallet_rpc.TrialTestWRPC_WS testMethod=test_notif>.client_factory

self       = <test_wallet_rpc.TrialTestWRPC_WS testMethod=test_notif>

jmclient/test/test_wallet_rpc.py:181: AssertionError
------------------------------------------------------- Captured stdout call -------------------------------------------------------
User data location: .
2022-09-16 14:56:12,583 [DEBUG]  rpc: getblockchaininfo []
2022-09-16 14:56:12,587 [DEBUG]  rpc: listwallets []
2022-09-16 14:56:12,589 [DEBUG]  rpc: getwalletinfo []
2022-09-16 14:56:12,850 [DEBUG]  rpc: getnewaddress []
2022-09-16 14:56:12,870 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
2022-09-16 14:56:12,934 [DEBUG]  rpc: sendtoaddress ['bcrt1q9u497juc35ks6x3u4ugpknyhytndz9jraav3rz', 2.0]
2022-09-16 14:56:14,288 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
2022-09-16 14:56:14,316 [DEBUG]  rpc: sendtoaddress ['bcrt1qh2yae8jgqe259kde3llqv3hwp4k9f8c6kk34s6', 2.0]
2022-09-16 14:56:15,540 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
2022-09-16 14:56:15,567 [DEBUG]  rpc: sendtoaddress ['bcrt1qx37es2nmve0epc9cvj94687z3amap7dlc0eh4w', 2.0]
2022-09-16 14:56:16,883 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
2022-09-16 14:56:16,900 [DEBUG]  rpc: sendtoaddress ['bcrt1qyqq7xgmsg6x3ldfdnqr2jz2wudt2xew5tkxn2a', 2.0]
2022-09-16 14:56:18,308 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
2022-09-16 14:56:18,319 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
2022-09-16 14:56:18,326 [DEBUG]  rpc: listaddressgroupings []
2022-09-16 14:56:18,942 [INFO]  Detected new wallet, performing initial import
2022-09-16 14:56:18,943 [DEBUG]  requesting detailed wallet history
2022-09-16 14:56:18,958 [DEBUG]  rpc: listlabels []
2022-09-16 14:56:19,032 [DEBUG]  rpc: listunspent [0]
2022-09-16 14:56:20,128 [DEBUG]  bitcoind sync_unspent took 1.0983452796936035sec
2022-09-16 14:56:20,148 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
2022-09-16 14:56:20,150 [WARNING]  Connection had broken pipe, attempting reconnect.
2022-09-16 14:56:20,159 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
2022-09-16 14:56:20,169 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
2022-09-16 14:56:20,176 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
2022-09-16 14:56:20,185 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
2022-09-16 14:56:20,196 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
2022-09-16 14:56:20,211 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
2022-09-16 14:56:20,224 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
2022-09-16 14:56:20,242 [DEBUG]  rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
-------------------------------------------------------- Captured log call ---------------------------------------------------------
DEBUG    joinmarket:blockchaininterface.py:232 rpc: getblockchaininfo []
DEBUG    joinmarket:blockchaininterface.py:232 rpc: listwallets []
DEBUG    joinmarket:blockchaininterface.py:232 rpc: getwalletinfo []
DEBUG    joinmarket:blockchaininterface.py:232 rpc: getnewaddress []
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: sendtoaddress ['bcrt1q9u497juc35ks6x3u4ugpknyhytndz9jraav3rz', 2.0]
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: sendtoaddress ['bcrt1qh2yae8jgqe259kde3llqv3hwp4k9f8c6kk34s6', 2.0]
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: sendtoaddress ['bcrt1qx37es2nmve0epc9cvj94687z3amap7dlc0eh4w', 2.0]
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: sendtoaddress ['bcrt1qyqq7xgmsg6x3ldfdnqr2jz2wudt2xew5tkxn2a', 2.0]
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: listaddressgroupings []
INFO     joinmarket:wallet_service.py:559 Detected new wallet, performing initial import
DEBUG    joinmarket:wallet_service.py:748 requesting detailed wallet history
DEBUG    joinmarket:blockchaininterface.py:232 rpc: listlabels []
DEBUG    joinmarket:blockchaininterface.py:232 rpc: listunspent [0]
DEBUG    joinmarket:wallet_service.py:846 bitcoind sync_unspent took 1.0983452796936035sec
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
WARNING  joinmarket:jsonrpc.py:113 Connection had broken pipe, attempting reconnect.
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q58r5yvy6m6m98lzxtuvzd97jmm0jjx6kmqjxn5']
DEBUG    joinmarket:blockchaininterface.py:232 rpc: generatetoaddress [1, 'bcrt1q9axcq6ujnzcxrael3dlp2qf20eeph2uktefr28']
```

However, unlike @kristapsk, I **was** able to get the test to pass by increasing the delays. However, adding delays to make a unit test pass is an anti-pattern and is not the correct answer here. The speed and load of the system should affect only how long a program takes to execute, not whether it executes correctly.

--- a/jmclient/test/test_wallet_rpc.py
+++ b/jmclient/test/test_wallet_rpc.py
@@ -169,7 +169,8 @@ class TrialTestWRPC_WS(WalletRPCTestBase, unittest.TestCase):
         # create a small delay between the instruction to send
         # the notification, and the checking of its receipt,
         # otherwise the client will be queried before the notification
-        # arrived:
+        # arrived. We will try a few times before giving up.
+        self.attempt_receipt_counter = 0
         d.addCallback(self.wait_to_receive)
         return d
 
@@ -178,7 +179,13 @@ class TrialTestWRPC_WS(WalletRPCTestBase, unittest.TestCase):
         return d
     
     def checkNotifs(self):
-        assert self.client_factory.notifs == 1
+        if self.attempt_receipt_counter > 10:
+            assert False
+        if not self.client_factory.notifs == 1:
+            jlog.info("Failed to receive notification, waiting and trying again")
+            self.attempt_receipt_counter += 1
+            d = task.deferLater(reactor, 0.2, self.checkNotifs)
+            return d
 
     def fire_tx_notif(self):
         self.daemon.wss_factory.sendTxNotification(self.test_tx,

